packages/text/README.md

```diff
@@ -0,0 +1,76 @@
+# hixme-ui text
+
+```bash
+npm i --save @hixme-ui/text
+```
+
+## Usage
+
+```javascript
+import Text, {
+ TextLink,
+ Paragraph,
+ Label,
+} from '@hixme-ui/text'
+
+// text success
+<Text success>I said good day sir!</Text>
+
+// paragraph warning
+<Text p warning>Are you sure?</Text>
+<Paragraph warning>Are you sure?</Paragraph>
+```
+
+## Props
+
+| Category        | Name            | Type        |
+---------------------------------------------------
```

Comment:
Adding pipes on this line should fix the table formatting for github

Reply:
thanks! done!